tests/integration/fixtures.py

```diff
@@ -14,6 +14,16 @@
 except ImportError:
     cudf = None
 
+try:
+    import dask_cudf
+except ImportError:
+    dask_cudf = None
+
+try:
+    from dask_cuda import LocalCUDACluster  # noqa: F401
+except ImportError:
```

Comment:
Nitpick but can these imports be placed in a specific test? Something like

```python
def test_jvm_segfault():
    import dask_cudf
    from dask_cuda import LocalCUDACluster
```

If they need to be imported here to trigger the segfault, then can they be consolidated with the above try-except block with a comment noting why they appear?

```python
try:
    import cudf
    # importing to check for JVM segfault
    import dask_cudf
    from dask_cuda import LocalCUDACluster
except ImportError:
    cudf = None
```

Reply:
@charlesbluca yes, they needed to be imported there to properly test for the failure. I will combine them however.